model/host/host.go

```diff
@@ -119,6 +119,11 @@ type Host struct {
 	DockerOptions DockerOptions `bson:"docker_options,omitempty" json:"docker_options,omitempty"`
 }
 
+type HostsByDistro struct {
+	DistroID string `bson:"distro_id" json:"distro_id" mapstructure:"distro_id"`
```

Comment:
Since you're just using this for a query, no need for the json and mapstructure tags

Reply:
Removed.